features/step_definitions/machine.rb

```diff
@@ -84,6 +84,16 @@
   cb["new_node"] = node_name
 end
 
+Then(/I check that cluster does not have empty publiczone$/) do
```

Comment:
The following line is `ensure_admin_tagged`, so `I check ...` better be `admin checks ...`